tests/conditional_processing/tests.py

```diff
@@ -188,7 +193,32 @@ def test_single_condition_head(self):
         response = self.client.head('/condition/')
         self.assertNotModified(response)
 
+    # Test that the same ETag results whether returning quoted
```

Comment:
Should these comments be docstrings? We're in the habit of stating the expected behavior rather than including "Test that..." since all tests test things.